examples/annotation_label_studio/materializers/fastai_learner_materializer.py

```diff
@@ -0,0 +1,62 @@
+import os
+import tempfile
+from typing import Type
+
+from fastai.learner import Learner
+from fastai.vision.all import *
+
+from zenml.artifacts import ModelArtifact
+from zenml.io import fileio
+from zenml.logger import get_logger
+from zenml.materializers.base_materializer import BaseMaterializer
+
+DEFAULT_MODEL_FILENAME = "model.pkl"
+
+logger = get_logger(__name__)
+
+
+class FastaiLearnerMaterializer(BaseMaterializer):
+    """Materializer to read/write fastai models."""
+
+    ASSOCIATED_TYPES = (Learner,)
+    ASSOCIATED_ARTIFACT_TYPES = (ModelArtifact,)
+
+    def handle_input(self, data_type: Type[Learner]) -> Learner:
+        """Reads and returns a fastai model.
+        Only loads the model, not the checkpoint.
+
+        Args:
+            data_type: The type of the model to load.
+        Returns:
+            A loaded fastai model.
+        """
+        super().handle_input(data_type)
+
+        def is_cat(x):
```

Comment:
What does this do?

Reply:
Not sure, but that will all be ironed out + we will no longer have all those duplicates lying around when it's done.